index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="responsive.css">
+    
+    <title>Document</title>
+</head>
+<body>
+  <div id="main">
+      <div id="title">
+          <h1>Nelson Mandela</h1>
+          <h5>The immortal leader of South Africa</h5>
+      </div>
+      <div id="img-div">

```

Comment:
Please consider using semantic html tags over divs. This ensures that your code is self-explanatory and anyone can relate the content of the block just by the tags enclosing it. For example, consider using a `<header>` tag for the titles.